testsuite/MDAnalysisTests/analysis/test_helix_analysis.py

```diff
@@ -430,6 +435,15 @@ def test_residue_gaps_no_split(self, psf_ca):
         assert 'has gaps in the residues' in warnmsg
         assert 'Splitting into' not in warnmsg
 
+    def test_len_groups_short(self, psf_ca):
+        sel = 'resnum 161-168'
+        with pytest.warns(UserWarning) as rec:
```

Comment:
As above, rather than checking rec, I suggest just using `match` to make sure you're picking up the right warning.